pxblue/angular-component-library
# Fix info list item wrap prop bug

Fixes #269.


#### Changes proposed in this Pull Request:
- Fixed info list item wrap prop bug for title and subtitle.


#### To Test:
- Open storybook 
- Turn on wrap title and wrap subtitle props via knobs
- Update title and subtitle to be one long word
- Observe word wrap
